numpy/ma/core.py

```diff
@@ -2081,6 +2081,10 @@ def masked_equal(x, value, copy=True):
     """
     Mask an array where equal to a given value.
 
+    Return a MaskedArray, masked where the data in array `x` are
+    equal to `value`. The fill_value of the returned MaskedArray
+    is set to `value`.
+
     This function is a shortcut to ``masked_where``, with
     `condition` = (x == value).  For floating point arrays,
     consider using ``masked_values(x, value)``.
```

Comment:
```suggestion
    For floating point arrays, consider using ``masked_values(x, value)``.
```

Reply:
No point in mentioning ``masked_where`` when it differs w.r.t. to the fill_value setting and is already mentioned in "see also"